scripts/openmc-make-depletion-chain-casl

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python3
+
+import glob
+import os
+from zipfile import ZipFile
+from collections import OrderedDict, defaultdict
+from io import StringIO
+from itertools import chain
+
+try:
+    import lxml.etree as ET
+    _have_lxml = True
+except ImportError:
+    import xml.etree.ElementTree as ET
+    _have_lxml = False
+
+import openmc.data
+import openmc.deplete
+from openmc._xml import clean_indentation
+from openmc.deplete.chain import _REACTIONS
+from openmc.deplete.nuclide import Nuclide, DecayTuple, ReactionTuple
+from openmc._utils import download
+
+from casl_chain import CASL_CHAIN
+
+URLS = [
+    'http://www.nndc.bnl.gov/endf/b7.1/zips/ENDF-B-VII.1-neutrons.zip',
+    'http://www.nndc.bnl.gov/endf/b7.1/zips/ENDF-B-VII.1-decay.zip',
+    'http://www.nndc.bnl.gov/endf/b7.1/zips/ENDF-B-VII.1-nfy.zip'
+]
+
+def main():
+    if os.path.isdir('./decay') and os.path.isdir('./nfy') and os.path.isdir('./neutrons'):
+        endf_dir = '.'
+    elif 'OPENMC_ENDF_DATA' in os.environ:
+        endf_dir = os.environ['OPENMC_ENDF_DATA']
+    else:
+        for url in URLS:
+            basename = download(url)
+            with ZipFile(basename, 'r') as zf:
+                print('Extracting {}...'.format(basename))
+                zf.extractall()
+        endf_dir = '.'
+
+    decay_files = glob.glob(os.path.join(endf_dir, 'decay', '*.endf'))
+    fpy_files = glob.glob(os.path.join(endf_dir, 'nfy', '*.endf'))
+    neutron_files = glob.glob(os.path.join(endf_dir, 'neutrons', '*.endf'))
+
+    # Create a Chain
+    chain = openmc.deplete.Chain()
+
+    print('Reading ENDF nuclear data from "{}"...'.format(os.path.abspath(endf_dir)))
+
+    # Create dictionary mapping target to filename
+    print('Processing neutron sub-library files...')
+    reactions = {}
+    for f in neutron_files:
+        evaluation = openmc.data.endf.Evaluation(f)
+        name = evaluation.gnd_name
+        if name in CASL_CHAIN:
+            reactions[name] = {}
+            for mf, mt, nc, mod in evaluation.reaction_list:
+                if mf == 3:
+                    file_obj = StringIO(evaluation.section[3, mt])
+                    openmc.data.endf.get_head_record(file_obj)
```

Comment:
Question: Just to confirm, since `get_head_record` calls `file_obj.readline()`, you *must* call `openmc.data.endf.get_head_record` here in order for `openmc.data.endf.get_cont_record` to output what you expect?  I'm going off the `openmc` readthedocs here [1].

If the above is not true, it looks like the return value of `get_head_record` goes unused.

[1] https://openmc.readthedocs.io/en/latest/_modules/openmc/data/endf.html